torch/testing/_internal/common_methods_invocations.py

```diff
@@ -623,6 +623,15 @@ def __init__(self,
                  assert_jit_shape_analysis=False,  # assert that jit shape analysis fully propagates shape
                  # the following metadata relates to ExpandedWeights support and is checked in test_expanded_weights.py
                  supports_expanded_weight=False,
+                 # FIXME: this should be changed to default to True
+                 # when all lambdas from torch/overrides have been ported to OpInfos
+                 # in order to throw errors that alert the user to fill in override_lambda
+                 supports_overrides=False,  # whether the op supports overriding
+                 override_lambda=None,  # a lambda function that has the same signature as the real function
```

Comment:
This comment is great, it might be easier to write above `override_lambda` instead of alongside it, though